contracts/LockedNORILib.sol

```diff
@@ -59,26 +60,26 @@ library ScheduleUtils {
     Schedule storage schedule,
     uint256 time,
     uint256 amount
-  ) internal {
+  ) public {
```

Comment:
@jaycenhorton This change from internal to public causes the linking error because it makes this no longer a purely functional library that can be interpolated into the using contract.